gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManager.java

```diff
@@ -796,7 +796,7 @@ private ContextAwareCounter getRunningJobsCounter(DagNode<JobExecutionPlan> dagN
 
     private List<ContextAwareCounter> getRunningJobsCounterForUser(DagNode<JobExecutionPlan> dagNode) {
       Properties props = dagNode.getValue().getJobSpec().getConfigAsProperties();
-      String proxy = props.getProperty("user.to.proxy");
+      String proxy = props.getProperty(AzkabanProjectConfig.USER_TO_PROXY);
       List<ContextAwareCounter> counters = new ArrayList<>();
 
       if (proxy != null) {
```

Comment:
Nit: Can this be replaced with if (!Strings.isNullOrEmpty(proxy)) similar to what you have in line 811?